dashboard/app/models/script_level.rb

```diff
@@ -59,15 +59,21 @@ def self.add_script_levels(script, lesson, raw_script_levels, counters, new_suff
       raw_script_level.symbolize_keys!
 
       properties = raw_script_level.delete(:properties) || {}
-
-      levels = Level.add_levels(raw_script_level[:levels], script, new_suffix, editor_experiment)
-
-      if new_suffix && properties[:variants]
-        properties[:variants] = properties[:variants].map do |old_level_name, value|
-          ["#{old_level_name}_#{new_suffix}", value]
-        end.to_h
+      raw_levels = raw_script_level[:levels]
+
+      # variants are deprecated. when cloning a script, retain only the first
+      # active level in each script level, discarding any variants.
+      if new_suffix && (raw_levels.length > 1 || properties[:variants])
```

Comment:
Is it possible for there to be multiple `raw_levels` but for `properties[:variants]` to be falsy?

Reply:
I don't think so... I'll simplify.